spec/helpers/topics_helper_spec.rb

```diff
@@ -3,22 +3,59 @@
 RSpec.describe TopicsHelper do
   describe "#document_type_count" do
     it "should return total counts for document types" do
-      mock_tagged_content = [
-        {
-        "format" => "publication",
-        "link" => "/government/publications/esfa-update-january-2017",
-        "title"=>"ESFA update: January 2017"
-      },
-      {
-        "format" => "publication",
-        "link" => "/government/publications/esfa-update-january-2018",
-        "title"=>"ESFA update: January 2018"
-      }
-    ]
+      mock_tagged_content = mocked_tagged_content
+
+      document_count = document_type_count(mock_tagged_content)
+
+      expect(document_count["publication"]).to eq(2)
+    end
+  end
+
+  describe "#group_tagged_content" do
+    it "should group the tagged content by document type" do
+      tagged_content = mocked_tagged_content
+
+      grouped_tagged_content = grouped_tagged_content(tagged_content)
+
+      expect(grouped_tagged_content.keys().length).to eq(3)
 
-    document_count = document_type_count(mock_tagged_content)
+      expected_groups = %w[publication news_article organisation].sort
+      expect(grouped_tagged_content.keys()).to include(*expected_groups)
 
-    expect(document_count["publication"]).to eq(2)
+      expect(grouped_tagged_content["news_article"]).to include(
+        {
+          "format" => "news_article",
+          "link" => "/government/news/education-bill-receives-royal-assent",
+          "title"=>"Education Bill receives Royal Assent"
+        }
+      )
+
+      expect(grouped_tagged_content.find_index { |k,_| k == "publication" }).to eq(0)
     end
   end
+
+  def mocked_tagged_content
```

Comment:
Should this go in a test helper file like our other tests?

Reply:
Good point. I'll add that in.